clients/fides-js/src/lib/hooks.ts

```diff
@@ -107,11 +107,14 @@ export const useConsentServed = ({
       }
       const request: RecordConsentServedRequest = {
         browser_identity: event.detail.identity,
-        privacy_experience_id: privacyExperience.id,
+        privacy_experience_id:
+          privacyExperience.experience_config?.translations[0]
+            .experience_config_history_id,
         user_geography: userGeography,
         acknowledge_mode: acknowledgeMode,
+        // TODO: pass in specific language shown in UI
         privacy_notice_history_ids: notices.map(
-          (n) => n.privacy_notice_history_id
+          (n: PrivacyNotice) => n.translations[0].privacy_notice_history_id
```

Comment:
Yeah... this is an interesting detail I hadn't noticed before... if the history ID needs to match the translated version shown in the UI that means this hook actually needs to _know_ which translation is selected... hmm.

Adding a TODO to #4615